packages/neos-ui/src/Containers/Modals/AddNode/step2.js

```diff
@@ -1,11 +1,12 @@
-import React, {PureComponent, PropTypes} from 'react';
+import React, {Component, PropTypes} from 'react';
 import {$get} from 'plow-js';
 import Button from '@neos-project/react-ui-components/lib/Button/';
 import Dialog from '@neos-project/react-ui-components/lib/Dialog/';
 import I18n from '@neos-project/neos-ui-i18n';
 import EditorEnvelope from '@neos-project/neos-ui-editors/src/EditorEnvelope/index';
+import style from './style.css';
 
-export default class Step2 extends PureComponent {
+export default class Step2 extends Component {
```

Comment:
Why do you inherit from Component, and not PureComponent? any particular issue you encountered?